library/log.inc

```diff
@@ -47,7 +47,7 @@ function newEvent($event, $user, $groupname, $success, $comments = "", $patient_
 
     // Send item to log_comment_encrypt for comment encyption tracking
     $last_log_id = $GLOBALS['adodb']['db']->Insert_ID();
-    $encryptLogQry = "INSERT INTO log_comment_encrypt (log_id, encrypt, checksum, `version`) ".
+    $encryptLogQry = "INSERT INTO log_comment_encrypt (log_id, encrypt, checksum, version) ".
```

Comment:
This should be ok. I generally always use backtick, although in this case I really did it since version is so generic (in case mysql ever decides to use it; back ticks protect column names from clashing with mysql terms)
I am guessing I messed up the sql upgrade script. Will look into it now.

Reply:
btw, here's a listing of the mysql keyword:
https://dev.mysql.com/doc/refman/5.7/en/keywords.html
(turns our version is not on there, but you never know; about 5 years ago they made a new keyword that did clash in some of openemr code; forgot the name though)